integrated_channels/integrated_channel/models.py

```diff
@@ -401,7 +401,28 @@ class ContentMetadataItemTransmission(TimeStampedModel):
         null=True,
     )
     deleted_at = models.DateTimeField(
-        help_text='Date when the content transmission was deleted',
+        help_text='Date when the content transmission was deleted in the remote API',
+        blank=True,
+        null=True
+    )
+    created_at = models.DateTimeField(
```

Comment:
before this refactor, the absence of a record meant it wasn't transmitted - either because an attempt hadn't yet been made or the attempt(s) failed. after this refactor, records are created before a transmission is sent so, we need to differentiate "new and unsent" from created records.